### PR TITLE
Onnx export

### PR DESCRIPTION
**Pull request by monkey0head**
targer_opset check added to convert_to_onnx_object

### CODE REVIEW

File: catboost/python-package/catboost/utils.py [hunk 1]
```diff
@@ -670,9 +670,10 @@ def convert_to_onnx_object(model, export_parameters=None, **kwargs):
             "There is no trained model to use save_model(). Use fit() to train model. Then use this method.")
 
     for kwarg in kwargs:
-        if kwarg in ['initial_types', 'target_opset', 'custom_conversion_functions', 'custom_shape_calculators'] \
-                and kwargs[kwarg] is not None:
-            warnings.warn(str(kwarg + " argument is not supported"))
+        if kwarg == 'target_opset' and kwargs[kwarg] is not None and kwargs[kwarg] != 2:
+            warnings.warn(str(kwarg + " argument is not supported. Default target_opset is 2 (ai.onnx.ml domain)"))
+        elif kwarg == 'initial_types' and kwargs[kwarg] is not None:
+                warnings.warn(str(kwarg + " argument is not supported"))
```
Comment (nikitxskv) [inline catboost/python-package/catboost/utils.py:676]:
```
for name, value in kwargs.items():
	if name == 'target_opset' and value not in [None, 2]:
		warnings.warn('target_opset argument is not supported. Default target_opset is 2 (ai.onnx.ml domain)')
	elif name == 'initial_types' and value is not None:
		warnings.warn('initial_types argument is not supported')
```

**Comment by nikitxskv** [comment]
@arcadia-devtools Ship it!

**Comment by arcadia-devtools** [comment]
CLA already signed

**Comment by arcadia-devtools** [comment]
@nikitxskv, internal review request created: [1275147](https://a.yandex-team.ru/review/1275147)

**Comment by arcadia-devtools** [comment]
Check [1275147](https://a.yandex-team.ru/review/1275147) passed

**Pull request merged**